app/vep/models/pipeline_model.py

```diff
@@ -1,55 +1,69 @@
 from typing import List
-from pydantic import BaseModel
-from core.config import NF_COMPUTE_ENV_ID, UPLOAD_DIRECTORY
+from pydantic import BaseModel, model_serializer
+from core.config import (
+    NF_COMPUTE_ENV_ID,
+    UPLOAD_DIRECTORY,
+    NF_PIPELINE_URL,
+    NF_WORK_DIR,
+)
 from core.logging import InterceptHandler
-import logging
+import logging, json, os
 
 logging.getLogger().handlers = [InterceptHandler()]
 
-import tempfile 
 
 class VEPConfigParams(BaseModel):
-  vcf: str
-  vep_config: str
+    vcf: str
+    vep_config: str
+
+    @model_serializer
+    def vep_config_serialiser(self):
+        vcf_str = '"vcf":"' + self.vcf + '"'
+        config_str = '"vep_config":"' + self.vep_config + '"'
+        stringified_encoded_json = "{" + vcf_str + "," + config_str + "}"
+        return stringified_encoded_json
+
 
 class LaunchParams(BaseModel):
-  computeEnvId: str = NF_COMPUTE_ENV_ID
-  pipeline: str
-  workDir: str
-  revision: str = "main"
-  pullLatest: bool = True
-  configProfiles: List[str] = ["slurmnew"]
-  paramsText: VEPConfigParams
-  preRunScript: str = "module load nextflow"
-  postRunScript: str = ""
-  stubRun: bool = False
-  labelIds: List[str]
-  headJobCpus: int = 1
-  headJobMemoryMb: int = 4
+    computeEnvId: str = NF_COMPUTE_ENV_ID
+    pipeline: str = NF_PIPELINE_URL
+    workDir: str = NF_WORK_DIR
+    revision: str = "main"
+    pullLatest: bool = True
+    configProfiles: List[str] = ["slurmnew"]
+    paramsText: VEPConfigParams
+    preRunScript: str = "module load nextflow"
+    postRunScript: str = ""
+    stubRun: bool = False
+    labelIds: List[str]
+    headJobCpus: int = 1
+    headJobMemoryMb: int = 4
+
 
 class PipelineParams(BaseModel):
-  launch: LaunchParams
+    launch: LaunchParams
+
 
 class ConfigIniParams(BaseModel):
-  cache: int = 1
-  dir_cache: str = "/nfs/production/flicek/ensembl/variation/data/VEP/tabixconverted/"
-  species: str = "homo_sapiens"
-  assembly: str = 'GRCh38'
-  cache_version: int = 110
-  offline: int = 1
-  force_overwrite: int = 1
-  symbol: bool = False
-  biotype: bool = False
-  transcript_version: int = 1
-  canonical: int = 1
+    cache: int = 1
+    dir_cache: str = "/nfs/production/flicek/ensembl/variation/data/VEP/tabixconverted/"
+    species: str = "homo_sapiens"
+    assembly: str = "GRCh38"
+    cache_version: int = 110
+    offline: int = 1
+    force_overwrite: int = 1
+    symbol: bool = False
+    biotype: bool = False
+    transcript_version: int = 1
+    canonical: int = 1
 
-  # Creates config ini file
-  def create_config_ini_file(self):
+    # Creates config ini file
```

Comment:
Doesn't the method name already tell what the method does? What's the value of this comment?